### ISSUE TITLE
Bug: File still listed as Modified even after push or fetch.

### ISSUE THREAD

**Issue by hg0428** [thebaselab/codeapp#451]
Files are still listed as modified even after I push the commit or I fetch the latest commits.

This is a very bad bug.

**Comment by bummoblizard** [comment]
Did you make sure that the changes are committed? You'll need to add the files to staged change and press the check button to make a commit.

**Comment by hg0428** [comment]
> Did you make sure that the changes are committed? You'll need to add the files to staged change and press the check button to make a commit.

Yes, I did, github says they were pushed and I was able to fetch the updates on replit.

**Comment by hg0428** [comment]
Another problem, I pushed 2 new commits from another service, and I am unable to fetch them on Code App. I waited 30 minutes and restarted the app a few times, and I can not seem to find a way to fetch the new commits.
This is all I see:
![34C32D29-54A4-4030-ADAB-CC66698A3501](https://user-images.githubusercontent.com/45984899/168677784-9f7b05b0-1c59-4a47-afe7-5974fb5d8360.jpeg)

**Comment by bummoblizard** [comment]
After fetching, you'll need to manually check out to origin/master.

**Comment by hg0428** [comment]
Ok, I got that working. There is definitely a lot of room for improvement in the UI and how this app functions. I am used to using replit's github integration. In the sidebar, it has a button for pushing a commit, and it tells you how many commits ahead of / behind it is, if it is behind, then is had a button to fetch.
It also has a dropdown menu for selecting branches with a button to create a new branch.

Even if you don't change the UI here, at least give a tour/give instructions on how it works.

**Comment by hg0428** [comment]
> After fetching, you'll need to manually check out to origin/master.

I have an update on this bug.
Maybe its my client, but when I fetch the latest commits (from another ide) after pushing from code app, no changes were made.
